### DISCUSSION TITLE
Leggere fatture dal cassetto fiscale

### DISCUSSION THREAD

**Discussion by pupiweb** [fattureincloud/api#449] [Q&A] [answered]
Buongiorno --
L'esigenza è quella di far leggere fatture attive e passive di N Ditte, ognuna con la propria P. IVA, ad una ditta "controllante" (con l'assenso delle "controllate" ovviamente)
Credo la cosa si possa fare ottenendo per ciascuna il token col quale interagire con le API, e ricevere la lista dei documenti emessi e ricevuti

La domanda è se questa lista di documenti includa *tutte* le fatture delle ditte, comunque inviate allo SDI o dallo SDI ricevute, o se invece la cosa funzioni *solo* con le fatture inviate tramite FIC

grazie

**Comment by valmoz** [comment, marked as answer]
Buongiorno @pupiweb,
quando una fattura viene inviata all'ADE essa viene inviata con un formato standard da essi fornito, quindi Fatture in Cloud riceverà tutte le fatture destinate all'azienda in questione, indipendentemente dal software che ha generato la fattura.

Le invio anche alcune guide non relative alle API che potrebbero essere utili:
- https://help.fattureincloud.it/help/articolo/7-comunica-codice-destinatario-agenzia-entrate
- https://help.fattureincloud.it/help/articolo/38-non-ricevo-fatture-note-credito-elettroniche-miei-fornitori

Confermo inoltre quanto da lei ipotizzato a riguardo dei token. 

A disposizione
Mauro

**Comment by pupiweb** [comment]
Intanto grazie :)

   Un'ultima verifica, ed una domanda "economica"
   1. La parte "riceverà tutte le fatture destinate all'azienda in questione" riguarda anche le fatture *emesse* dall'azienda ed inviate allo SDI/AdE con altri sistemi ?
   Come accennavo le varie ditte hanno già sistemi di fatturazione, che non posso modificare, per cui è importante poter leggere cosa hanno fatturato
   2. Vorrei proporre la cosa al cliente "chiavi in mano",  facendomi quindi carico dei costi di iscrizione a Fatture in Cloud delle varie ditte "controllate" (un centinaio), fermo restando che ogni ditta dovrà iscriversi autonomamente
   E' una cosa fattibile ?
   Grazie, attendo vostre
   g. pupita
   BibiSoftware
   ---------- Messaggio originale ----------

   Buongiorno @pupiweb,
   quando una fattura viene inviata all'ADE essa viene inviata con un formato standard da essi fornito, quindi Fatture in Cloud riceverà tutte le fatture destinate all'azienda in questione, indipendentemente dal software che ha generato la fattura.

   Le invio anche alcune guide non relative alle API che potrebbero essere utili:
     * https://help.fattureincloud.it/help/articolo/7-comunica-codice-destinatario-agenzia-entrate
     * https://help.fattureincloud.it/help/articolo/38-non-ricevo-fatture-note-credito-elettroniche-miei-fornitori

   Confermo inoltre quanto da lei ipotizzato a riguardo dei token.

   A disposizione
   Mauro

**Reply by valmoz** [reply]
Buongiorno @pupiweb 
no. Per quelle emesse dalle vostre aziende su altri sistemi non effettuiamo un download automatico dall'ADE.
Può eventualmente creare fatture tramite API, in modo tale da riportarle anche in Fatture in Cloud, facendo però attenzione a non inviale al SDI.
Le inoltro una guida che potrebbe esserle utile: https://help.fattureincloud.it/help/articolo/541-importa-fatture-anagrafiche-altri-documenti-altri-software.

Per quanto riguarda la seconda domanda, le consiglio di contattare direttamente il Supporto Clienti a info@fattureincloud.it, questa è solo una community per agevolare l'utilizzo tecnico delle nostre API, quindi non saprei risponderle.

A disposizione
Mauro

**Reply by valmoz** [reply]
Buongiorno @pupiweb 
possiamo chiudere questa discussione?

Grazie
Mauro

**Comment by pupiweb** [comment]
Si, certo, credevo fosse già chiusa

   Grazie delle informazioni, precise e puntuali

   g.
   ---------- Messaggio originale ----------

   Buongiorno @pupiweb
   possiamo chiudere questa discussione?

   Grazie
   Mauro
